### PR TITLE
circleCI: build docker image with latest gometalinter release (v2)

### PR DESCRIPTION
**Pull request by nhoriguchi**
This is updated version of #54, with some refactoring applied. Original issue is #52.

### CODE REVIEW

File: .circleci/images/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,12 @@ RUN SGX_SDK_URL="https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/sgx_l
   && rm -f /tmp/sgx_linux_x64_sdk.bin \
   && sudo ln -s /opt/intel/sgxsdk/environment /etc/profile.d/intel-sgxsdk.sh
 
-RUN GOMETALINTER_URL="https://github.com/alecthomas/gometalinter/releases/download/v2.0.11/gometalinter-2.0.11-linux-amd64.tar.gz" \
+RUN sudo apt-get install -y \
+  jq \
```
Comment (ynamiki) [inline .circleci/images/Dockerfile:22]: Please add 2 spaces to the indent (like line 13). This style is inherited from the [base](https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/buildpack-deps/images/bionic-curl/Dockerfile).

File: .circleci/images/Dockerfile [hunk 2]
```diff
@@ -18,7 +18,12 @@ RUN SGX_SDK_URL="https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/sgx_l
...
+  && GOMETALINTER_VERSION="$(curl -s https://api.github.com/repos/alecthomas/gometalinter/releases/latest | jq -r '.tag_name')" \
+  && echo "Latest Gometalinter version: $GOMETALINTER_VERSION" \
+  && GOMETALINTER_URL="https://github.com/alecthomas/gometalinter/releases/download/${GOMETALINTER_VERSION}/gometalinter-${GOMETALINTER_VERSION##v}-linux-amd64.tar.gz" \
```
Comment (ynamiki) [inline .circleci/images/Dockerfile:25]: FYI: I found another way:
```
GOMETALINTER_URL=$(curl https://api.github.com/repos/alecthomas/gometalinter/releases/latest \
| jq -r '.assets[].browser_download_url | select(test("-linux-amd64.tar.gz$"))')
```
But I think your code is more readable....

**Review by ynamiki — CHANGES_REQUESTED** [review]

File: .circleci/images/Dockerfile (hunk 2 above)
Comment (nhoriguchi) [reply .circleci/images/Dockerfile:25]: I believe that using `browser_download_url` is more robust, so I like it, thanks!

**Comment by nhoriguchi** [comment]
Updated with #58, so let's close it.

**Pull request closed without merging**